src/components/claim-modal.js

```diff
@@ -78,142 +71,110 @@ const ClaimModal = ({
+    const claimStatus = contract.methods.claimStatus(drizzleState.account, 0, 12).call();
 
     //
 
-    claimStatus.then(r => setClaimStatus(r))
-  }, [drizzleState.account, drizzleState.web3.networkId, modalState])
+    claimStatus.then((r) => setClaimStatus(r));
+  }, [drizzleState.account, drizzleState.web3.networkId, modalState]);
 
   const handleClaim = () => {
-    setModalState(1)
+    setModalState(1);
 
-    const tx = claimWeeks(claims)
-    tx.on('transactionHash', function(hash) {
-      setTxHash(hash)
-    })
+    const tx = claimWeeks(claims);
+    tx.on("transactionHash", function (hash) {
+      setTxHash(hash);
+    });
 
-    tx.then(handleClaimed).catch(err => {
-      setModalState(0)
-    })
-  }
+    tx.then(handleClaimed).catch(() => {
+      setModalState(0);
+    });
+  };
 
   const handleClaimed = () => {
-    setModalState(2)
-  }
+    setModalState(2);
+  };
 
   const handleCancel = () => {
-    setModalState(0)
-    onCancel()
-  }
+    setModalState(0);
+    onCancel();
+  };
 
-  const getTotalClaimable = claims => {
+  const getTotalClaimable = (claims) => {
     const unclaimedItems = claims
       .filter((claim, index) => Boolean(claimStatus[index]) === false)
-      .map(claim => drizzle.web3.utils.toBN(claim ? claim.value.hex : '0x0'))
+      .map((claim) => drizzle.web3.utils.toBN(claim ? claim.value.hex : "0x0"));
 
-    let totalClaimable
+    let totalClaimable;
 
     if (unclaimedItems.length > 0) {
-      totalClaimable = unclaimedItems.reduce(function(
-        accumulator,
-        currentValue,
-        currentIndex,
-        array
-      ) {
-        return accumulator.add(currentValue)
-      })
-    } else totalClaimable = '0'
-    return totalClaimable
-  }
-  const getTotalRewarded = claims =>
+      totalClaimable = unclaimedItems.reduce(function (accumulator, currentValue) {
+        return accumulator.add(currentValue);
+      });
+    } else totalClaimable = "0";
+    return totalClaimable;
+  };
+  const getTotalRewarded = (claims) =>
     claims
-      .map(claim => drizzle.web3.utils.toBN(claim ? claim.value.hex : '0x0'))
-      .reduce(function(accumulator, currentValue, currentIndex, array) {
-        return accumulator.add(currentValue)
-      })
+      .map((claim) => drizzle.web3.utils.toBN(claim ? claim.value.hex : "0x0"))
+      .reduce(function (accumulator, currentValue) {
+        return accumulator.add(currentValue);
+      });
 
-  const claimWeeks = claims => {
-    const contract = new Web3.eth.Contract(
-      MerkleRedeem.abi,
-      CONTRACT_ADDRESSES[drizzleState.web3.networkId]
-    )
-    const args = claimObjects(claims).filter(
-      (_claim, index) => Boolean(claimStatus[index]) === false
-    )
+  const claimWeeks = (claims) => {
+    const contract = new Web3.eth.Contract(MerkleRedeem.abi, CONTRACT_ADDRESSES[drizzleState.web3.networkId]);
+    const args = claimObjects(claims).filter((_claim) => Boolean(claimStatus[_claim.week]) === false);
```

Comment:
This is the actual fix, the rest is linter.

Before: 
`const args = claimObjects(claims).filter((_claim, index) => Boolean(claimStatus[index]) === false);`
After:
`const args = claimObjects(claims).filter((_claim) => Boolean(claimStatus[_claim.week]) === false);`